bin/setup

```diff
@@ -1,29 +1,36 @@
 #!/usr/bin/env ruby
+# frozen_string_literal: true
```

Comment:
This isn't strictly necessary, and rubocop will look for it on the first line of the file anyhow... so this can be removed

Reply:
Removed